.flake8

```diff
@@ -5,9 +5,12 @@
 # E402: module level import not at top of file
 # C901: too complex
 # W503: line break before binary operator
+# W504: line break after binary operator
+# F632: use ==/!= to compare str, bytes, and int literals
 exclude = venv*,__pycache__,node_modules,bower_components
-ignore = D203,W503
+ignore = D203,W503,W504
 max-complexity = 12
 max-line-length = 120
 per-file-ignores =
     **/__init__.py : F401
+    dmutils/api_stubs.py: F632
```

Comment:
Where is this rule exactly? (We're going to deprecate these old api_stubs so I'm curious)

Reply:
[Starts on line 213](https://github.com/alphagov/digitalmarketplace-utils/blob/7a947932a68d292b025944c1319a813610914105/dmutils/api_stubs.py#L213)

Reply:
[Starts here](https://github.com/alphagov/digitalmarketplace-utils/blob/7a947932a68d292b025944c1319a813610914105/dmutils/api_stubs.py#L213)

Reply:
I think we should fix those warnings rather than ignore the rule.

Reply:
Done :+1: